frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualPaymentResponse.tsx

```diff
@@ -0,0 +1,71 @@
+import { Stack, Text } from '@chakra-ui/react'
+
+import { useIndividualPaymentSubmission } from './queries'
+
+export const IndividualPaymentResponse = (submissionId: {
+  submissionId: string
+}): JSX.Element => {
+  if (!submissionId) throw new Error('Missing submissionId')
+
+  // TODO: get paymentData from IndividualResponsePage through submission endpoint
+  const { data: paymentData } = useIndividualPaymentSubmission()
```

Comment:
now I'm wondering if it makes sense to have the error and isLoading states (and the LoadingDecryption or just a normal loading skeleton) here too.. :sweat_smile: haha what do you think?

Reply:
Oh but then we will need to make a separate loadingdecrpytion skeleton for the payment response too right? Else it will look a lil awkward hahaha

Though I think it might be worth doing it, at least with a payment header or something, so the admin who sees it knows that he has payment enabled. Thanks @wanlingt will add it in!

Reply:
yup! or maybe can just use the standard Skeleton component from chakra not sure will have to consult @staceytan1998 !

thanks chifa :)

Reply:
[00473ba](https://github.com/opengovsg/FormSG/pull/5861/commits/00473bac19938dd74884715d2354f1b16e302bc3)